scripts/v1beta1/push.sh

```diff
@@ -36,72 +36,75 @@ echo "Image tag: ${TAG}"
 
 # Katib core images
 echo -e "\nPushing Katib controller image...\n"
-docker push ${REGISTRY}/katib-controller:${TAG}
+docker push "${REGISTRY}/katib-controller:${TAG}"
```

Comment:
Is it necessary to have double quotes here ?

Reply:
I got some warnings from [`shellcheck`](https://github.com/koalaman/shellcheck) and modified it.
Should I remove double quotes?

```shell
Line 39:
docker push ${REGISTRY}/katib-controller:${TAG}
            ^-- SC2086 (info): Double quote to prevent globbing and word splitting.
                                         ^-- SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
docker push "${REGISTRY}"/katib-controller:"${TAG}"
 
Line 42:
docker push ${REGISTRY}/katib-db-manager:${TAG}
            ^-- SC2086 (info): Double quote to prevent globbing and word splitting.
                                         ^-- SC2086 (info): Double quote to prevent globbing and word splitting.
...
```

Reply:
Sounds good, thanks!
I think we can add `shellcheck` to the CI check in the future.

Reply:
> I think we can add shellcheck to the CI check in the future.

Sounds good.
I created an [issue](https://github.com/kubeflow/katib/issues/1734) to resolve this.